rudder-server-root/SPECS/rudder-server-root.spec

```diff
@@ -52,7 +52,7 @@ Source8: rudder-server-root
 BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
 BuildArch: noarch
 
-Requires: rudder-jetty, rudder-webapp, rudder-inventory-endpoint, rudder-inventory-ldap, rudder-reports, rudder-agent, curl
+Requires: rudder-jetty, rudder-webapp, rudder-inventory-endpoint, rudder-inventory-ldap, rudder-reports, Rudder::Agent, curl
```

Comment:
Will rpm automatically install rudder-agent if it's not already there with this dependency ?
For example, on debian you should add an "or" with a real package here.